apps/dashboard/app/views/layouts/nav/_styles.html.erb

```diff
@@ -1,23 +1,24 @@
 <style>
+.navbar-color {
+  background-color: <%= Configuration.brand_bg_color || "#334155" %> !important;
+}
+
 <% if bg_color %>
 .navbar-inverse {
-  background-color: <%= bg_color %>;
-}
-.navbar-color {
-  background-color: <%= Configuration.brand_bg_color || "#334155" %>;
+  background-color: <%= bg_color %> !important;
 }
 <% end %>
-
 <% if link_active_color %>
-.navbar-inverse {
-  border-color: <%= link_active_color %>;
+.navbar-inverse .navbar-nav > .active > a, .navbar-inverse .navbar-nav > .active > a:hover, .navbar-inverse .navbar-nav > .active > a:focus {
+  background-color: <%= link_active_color %>;
 }
 
-.navbar-inverse .navbar-nav > .active > a, .navbar-inverse .navbar-nav > .active > a:hover, .navbar-inverse .navbar-nav > .active > a:focus {
+#navbar > ul.navbar-nav > li.nav-item > a:hover {
```

Comment:
I think we only want to display this when you click, not on hover

Reply:
Or when the menu is expanded in other words - that is the behavior in Bootstrap 3.

Reply:
Alternative: comment out the use of `link_active_color` since this is not used by default in bootstrap 4, and we can open a ticket to revisit later.